docs/examples/compute/instance_chef/README.md

```diff
@@ -0,0 +1,57 @@
+    #     ___  ____     _    ____ _     _____
+    #    / _ \|  _ \   / \  / ___| |   | ____|
+    #   | | | | |_) | / _ \| |   | |   |  _|
+    #   | |_| |  _ < / ___ | |___| |___| |___
+    #    \___/|_| \_/_/   \_\____|_____|_____|
+***
+## Manage an instance
+This example launches an instance into an existing subnet, registers the instance with a Chef server, includes a user-data script in the instance launch, remote executes a command, and outputs the public and private IP address of the instance.
+
+### Prerequisites
```

Comment:
How berks as a prereq, along with the setup to communicate with your server. (Does berks come with Chef DK?)

Also, knife must be installed and configured and in your path in order for the "knife node delete" to work on destroy.

Reply:
Berks is included with ChefDK according to https://docs.chef.io/berkshelf.html
I'll add a note about knife to the README.